docker-compose.yml

```diff
@@ -29,6 +29,9 @@ services:
       REDIS__CONNECTIONSTRING: ${REDIS__CONNECTIONSTRING:-redis:6379,password=password}
       TICKETSEARCH__SEARCHTYPE: ${TICKETSEARCH__SEARCHTYPE:-Mock}
       TICKETSTORAGE: ${TICKETSTORAGE:-ObjectStore}
+      JWT__AUTHORITY: ${JWT__AUTHORITY:-https://idtest.gov.bc.ca/login}     
+      JWT__AUDIENCE: ${JWT__AUDIENCE:-ca.bc.gov.ag.tco.dev}
+      OAuth__USERINFOENDPOINT: ${OAuth__USERINFOENDPOINT:-https://idtest.gov.bc.ca/oauth2/userinfo}
```

Comment:
Is this non-standard? Can it not be derived from `JWT__AUTHORITY` parameter? Less configuration is better.

Reply:
it is different in dev and prod